go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/kwilteam/kwil-db
 
-go 1.22
+go 1.22.7
+
+toolchain go1.23.3
```

Comment:
I have no clue why it's doing this and I can't seem to make it go away.

Reply:
Should we try to resolve or just let it be?